.github/workflows/nightly.yml

```diff
@@ -33,3 +33,12 @@ jobs:
         env:
             SSH_AUTH_SOCK: /tmp/ssh_agent.sock
         run: tox -e nightly  # Run tox using latest master branch from py42/c42eventextractor
+      - name: Notify Slack Action
+        uses: ravsamhq/notify-slack-action@v1.1
+        if: always()
+        with:
+          status: ${{ job.status }}
+          notify_when: 'success,failure,warnings'
```

Comment:
can we make this only post on failure? In general I think its a good idea to only put something in the alerts channel when action is needed.

Reply:
Yeah I just added the other two statuses for testing purposes, to avoid needing to trigger a failure to verify the connection works. Will remove.

Reply:
We are using a different action now, but it is configured to only post to Slack in the event of an action failure.